cmd/main.go

```diff
@@ -69,7 +69,10 @@ var (
 	setupLog = ctrl.Log.WithName("setup")
 )
 
-const memberClientTimeout = 3 * time.Second
+const (
+	memberClientTimeout = 3 * time.Second
+	userTierRootDir     = "templates/usertiers"
```

Comment:
let's put this into the `usertier_generator.go` file to reuse it also in the `usertier_generator_whitebox_test.go`